src/foam/dao/FixedSizeDAO.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * @license
+ * Copyright 2018 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.dao',
+  name: 'FixedSizeDAO',
+  extends: 'foam.dao.ProxyDAO',
+
+  documentation: function () {/*
```

Comment:
Just use a String rather than a function comment or a multi-line string (quoted with back-quotes "`") if it spans more than one line.